web-assembly/index.html

```diff
@@ -0,0 +1,63 @@
+---
+feature_name: Web Assembly
+chrome_version: 57
+feature_id: 5453022515691520
+---
+
+<h3>Background</h3>
+<p>Web Assembly is a binary code format for the web.</p>
+
+<script>
+  function log(id, ...args) {
+    const el = document.getElementById(id);
+
+    if (!el) {
```

Comment:
If you move the `log` function at the very bottom, you shouldn't need this check I believe.